src/client/app/suche/suche.component.ts

```diff
@@ -1445,25 +1446,41 @@ export class SucheComponent implements OnInit, AfterViewChecked {
                               poem.reservedPointer = this.allSearchResults.length;
                               console.log(poem.reservedPointer);
                               console.log(poem.value['5']);
-                              this.allSearchResults[ poem.reservedPointer ] = [];
-                                this.allSearchResults[ poem.reservedPointer ] = [
-                                  poem.value['8'], // Titel
-                                  poem.value['5'], // date
-                                  poem.value['7'], // text
-                                  poem.value['6'],
-                                  undefined,
-                                  5,
-                                  this.suchmaskeKonvolutIRIMapping[ k ].konvolut,
-                                  this.suchmaskeKonvolutIRIMapping[ k ].officialName,
-                                  8,
-                                  poem.value['11'],
-                                  10,
-                                  11,
-                                  poem.value['12'],
-                                  13,
-                                  poem.value['13']
-                                ];
 
+                              this.allSearchResults[ poem.reservedPointer ] = new CachePoem();
+                              this.allSearchResults[ poem.reservedPointer ].poemTitle = poem.value['8'];
+                              this.allSearchResults[ poem.reservedPointer ].poemCreationDate = poem.value['5'];
+                              this.allSearchResults[ poem.reservedPointer ].poemText = poem.value['7'];
+                              this.allSearchResults[ poem.reservedPointer ].poemIRI = poem.value['6'];
+                              this.allSearchResults[ poem.reservedPointer ].synopsisIRI = poem.value['11'];
+                              this.allSearchResults[ poem.reservedPointer ].synopsisTitle = poem.value['12'];
+                              this.allSearchResults[ poem.reservedPointer ].isFinalVersion = poem.value['13'];
+                              this.allSearchResults[ poem.reservedPointer ].searchConvolute
+                                = this.suchmaskeKonvolutIRIMapping[ k ].konvolut;
+                              this.allSearchResults[ poem.reservedPointer ].searchOfficialName
+                                = this.suchmaskeKonvolutIRIMapping[ k ].officialName;
+
+
+                            /*  old textgrid input data. no longer needed
```

Comment:
Na, wenn wir's nicht mehr brauchen, dann löschen! :-)

Reply:
Natürlich. Wir haben ja alles in der History.

Reply:
Der ursprüngliche Grund es zu behalten, war glaub's, dass die Zuordnung zum alten Index noch irgendwie möglich ist.